examples/Atlas/+atlasControllers/@InstantaneousQPController/setupAndSolveQP.m

```diff
@@ -20,7 +20,8 @@
 
 for j = 1:length(qp_input.body_motion_data)
   body_id = qp_input.body_motion_data(j).body_id;
-  [body_des, body_v_des, body_vdot_des] = evalCubicSplineSegment(t - qp_input.body_motion_data(j).ts(1), qp_input.body_motion_data(j).coefs);
+  t_spline = max(qp_input.body_motion_data(j).ts(1), min(qp_input.body_motion_data(j).ts(2), t));
```

Comment:
PlanEval always sends `ts` as a 2-element vector, since it is only sending exactly one piece of the spline polynomial, and that length is intentionally restricted by the LCM interface. We may eventually want to support longer vectors, but that will require some more logic in the controller to figure out which part of the polynomial is active. So, for now, `ts(2)` is correct.